x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/ml/datafeed_decreased_throughput_jsbase.json

```diff
@@ -0,0 +1,36 @@
+{
+  "job_id": "JOB_ID",
+  "indices": [
+    "INDEX_PATTERN_NAME"
+  ],
+  "max_empty_searches": 10,
+  "query": {
+    "bool": {
+      "filter": {
+        "term": {
+          "agent.name": "js-base"
+        }
+      }
+    }
+  },
+  "aggregations": {
+    "buckets": {
+      "date_histogram": {
+        "field": "@timestamp",
+        "fixed_interval": "900000ms"
+      },
+      "aggregations": {
+        "@timestamp": {
+          "max": {
+            "field": "@timestamp"
+          }
+        }
+      }
+    }
+  },
+  "scroll_size": 1000,
```

Comment:
We normally omit the `scroll_size` and `chunking_config` parts out of the datafeeds we ship with the modules. Are these being included deliberately?

Reply:
summoning @blaklaybul

Reply:
We can remove these.

Reply:
Removed in https://github.com/elastic/kibana/pull/53792/commits/ebe6353130f170600b92e45be911f70264e4f4be